<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>richTextEditor</class>
 <widget class="QWidget" name="richTextEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>296</width>
    <height>94</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="2">
    <widget class="QSplitter" name="splitter_3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="opaqueResize">
      <bool>false</bool>
     </property>
     <property name="childrenCollapsible">
      <bool>false</bool>
     </property>
     <widget class="QToolButton" name="alignLeft">
      <property name="toolTip">
       <string>Align Left</string>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/Graphics/alignLeft</normaloff>:/Graphics/alignLeft</iconset>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QToolButton" name="alignCenter">
      <property name="toolTip">
       <string>Align Center</string>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/Graphics/alignCenter</normaloff>:/Graphics/alignCenter</iconset>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QToolButton" name="alignRight">
      <property name="toolTip">
       <string>Align Right</string>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/Graphics/alignRight</normaloff>:/Graphics/alignRight</iconset>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QSplitter" name="splitter_4">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="opaqueResize">
      <bool>false</bool>
     </property>
     <property name="childrenCollapsible">
      <bool>false</bool>
     </property>
     <widget class="colorButton" name="textColor">
      <property name="toolTip">
       <string>Text Color</string>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="colorButton" name="backgroundColor">
      <property name="toolTip">
       <string>Background Color</string>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QSplitter" name="splitter_5">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="opaqueResize">
      <bool>false</bool>
     </property>
     <property name="childrenCollapsible">
      <bool>false</bool>
     </property>
     <widget class="QToolButton" name="undo">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="toolTip">
       <string>Undo</string>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/Graphics/undo</normaloff>:/Graphics/undo</iconset>
      </property>
      <property name="shortcut">
       <string>Ctrl+Z</string>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QToolButton" name="redo">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="toolTip">
       <string>Redo</string>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/Graphics/redo</normaloff>:/Graphics/redo</iconset>
      </property>
      <property name="shortcut">
       <string>Ctrl+Y</string>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QSplitter" name="splitter_6">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="opaqueResize">
      <bool>false</bool>
     </property>
     <property name="childrenCollapsible">
      <bool>false</bool>
     </property>
     <widget class="QToolButton" name="clearFormat">
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Clear Formatting[ALT+R]&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/Graphics/brush</normaloff>:/Graphics/brush</iconset>
      </property>
      <property name="shortcut">
       <string>Alt+R</string>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QToolButton" name="clear">
      <property name="toolTip">
       <string>Delete Selected Text</string>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/Graphics/clear</normaloff>:/Graphics/clear</iconset>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QToolButton" name="SelectAll">
      <property name="toolTip">
       <string>Select All</string>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/Graphics/selectall</normaloff>:/Graphics/selectall</iconset>
      </property>
      <property name="shortcut">
       <string>Ctrl+A</string>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QSplitter" name="splitter_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="opaqueResize">
      <bool>false</bool>
     </property>
     <property name="childrenCollapsible">
      <bool>false</bool>
     </property>
     <widget class="QToolButton" name="cut">
      <property name="toolTip">
       <string>Cut [CTRL+X]</string>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/Graphics/cut</normaloff>:/Graphics/cut</iconset>
      </property>
      <property name="shortcut">
       <string>Ctrl+X</string>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QToolButton" name="copy">
      <property name="toolTip">
       <string>Copy [CTRL+C]</string>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/Graphics/copy</normaloff>:/Graphics/copy</iconset>
      </property>
      <property name="shortcut">
       <string>Ctrl+C</string>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QToolButton" name="paste">
      <property name="toolTip">
       <string>Paste [CTRL+V]</string>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/Graphics/paste</normaloff>:/Graphics/paste</iconset>
      </property>
      <property name="shortcut">
       <string>Ctrl+V</string>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QToolButton" name="toUpper">
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Toggle selected text upper / lower case letters. [ALT+C]&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/Graphics/capsLock</normaloff>:/Graphics/capsLock</iconset>
      </property>
      <property name="shortcut">
       <string>Alt+C</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QSplitter" name="splitter">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="opaqueResize">
      <bool>false</bool>
     </property>
     <property name="childrenCollapsible">
      <bool>false</bool>
     </property>
     <widget class="QToolButton" name="bold">
      <property name="toolTip">
       <string>Bold [ALT+B]</string>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/Graphics/bold</normaloff>:/Graphics/bold</iconset>
      </property>
      <property name="shortcut">
       <string>Alt+B</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QToolButton" name="italic">
      <property name="toolTip">
       <string>Italic [ALT+I]</string>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/Graphics/italic</normaloff>:/Graphics/italic</iconset>
      </property>
      <property name="shortcut">
       <string>Alt+I</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QToolButton" name="underline">
      <property name="toolTip">
       <string>Underline [ALT+U]</string>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/Graphics/underline</normaloff>:/Graphics/underline</iconset>
      </property>
      <property name="shortcut">
       <string>Alt+U</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QToolButton" name="strike">
      <property name="toolTip">
       <string>Strike [ALT+S]</string>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/Graphics/strike</normaloff>:/Graphics/strike</iconset>
      </property>
      <property name="shortcut">
       <string>Alt+S</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QToolButton" name="highlight">
      <property name="toolTip">
       <string>Highlight [ALT+H]</string>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/Graphics/highlight</normaloff>:/Graphics/highlight</iconset>
      </property>
      <property name="shortcut">
       <string>Alt+H</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QComboBox" name="point">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <item>
      <property name="text">
       <string>8</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>9</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>10</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>11</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>12</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>14</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>16</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>18</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>20</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>22</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>24</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>36</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QFontComboBox" name="fontComboBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>250</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="currentFont">
      <font>
       <family>Tahoma</family>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QToolButton" name="serachOnline">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Search selected on scholar.Google.com [ALT+G]&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/ionicons/earth</normaloff>:/ionicons/earth</iconset>
     </property>
     <property name="shortcut">
      <string>Alt+G</string>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>colorButton</class>
   <extends>QToolButton</extends>
   <header>colorbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
